Wrappers/Python/ccpi/web_viewer/dev-environment.yml

```diff
@@ -1,10 +1,14 @@
 # run: conda env create --file environment.yml
 name: cilviewer_webapp
+channels:
+  - conda-forge
+  - ccpi
 dependencies:
 - python==3.9
 - matplotlib # Optional for more colormaps
 - h5py
 - pip
+- ccpi::cil-data
```

Comment:
does this force installing from the ccpi channel? If so, that is very useful!!

Reply:
Yes! it should do that